core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -1,8 +1,218 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { DotListingDataTableComponent } from '@components/dot-listing-data-table/dot-listing-data-table.component';
+import { MenuItem } from 'primeng/api';
+import { pluck, take, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { DotMessageService } from '@services/dot-message/dot-messages.service';
+import { DotAlertConfirmService } from '@services/dot-alert-confirm';
+import { DotPushPublishDialogService } from '@dotcms/dotcms-js';
+import { DotRouterService } from '@services/dot-router/dot-router.service';
+import { ActionHeaderOptions } from '@models/action-header';
+import { DataTableColumn } from '@models/data-table';
+import { DotContainer } from '@models/container/dot-container.model';
+import { DotContentState } from '@dotcms/dotcms-models';
+import {
+    DotContainerListState,
+    DotContainerListStore
+} from '@portlets/dot-containers/container-list/store/dot-container-list.store';
 
 @Component({
     selector: 'dot-container-list',
     templateUrl: './container-list.component.html',
-    styleUrls: ['./container-list.component.scss']
+    styleUrls: ['./container-list.component.scss'],
+    providers: [DotContainerListStore]
 })
-export class ContainerListComponent {}
+export class ContainerListComponent implements OnInit, OnDestroy {
+    @ViewChild('listing', { static: false })
+    vm$ = this.store.vm$;
+
+    listing: DotListingDataTableComponent;
+    containerBulkActions: MenuItem[];
+    selectedContainers: DotContainer[] = [];
+    addToBundleIdentifier: string;
+    actionHeaderOptions: ActionHeaderOptions;
+    tableColumns: DataTableColumn[];
+
+    private isEnterPrise: boolean;
+    private hasEnvironments: boolean;
+    private destroy$: Subject<boolean> = new Subject<boolean>();
+
+    constructor(
+        private store: DotContainerListStore,
+        private route: ActivatedRoute,
```

Comment:
Since the data is coming now from the CS, this should be move into store as well.

Reply:
updated